src/com/lilithsthrone/game/character/GameCharacter.java

```diff
@@ -5336,20 +5336,30 @@ public String getCompanionSexRejectionReason(boolean companionIsSub) {
 
 	protected Set<GameCharacter> getChildren() {
 		HashSet<GameCharacter> result = new HashSet<>();
-		getLittersBirthed().stream()
-			.flatMap(l -> l.getOffspring().stream().map(o -> {
-				try { return Main.game.getNPCById(o); }
-				catch(Exception e) { return null; }
-			}))
-			.filter(Objects::nonNull)
-			.forEach(result::add);
-		getLittersFathered().stream()
-			.flatMap(l -> l.getOffspring().stream().map(o -> {
-				try { return Main.game.getNPCById(o); }
-				catch(Exception e) { return null; }
-			}))
-			.filter(Objects::nonNull)
-			.forEach(result::add);
+		GameCharacter gc;
+
+		for(Litter litter : getLittersBirthed()) {
+			for(String id : litter.getOffspring()) {
+				try {
+					gc = Main.game.getNPCById(id);
+				} catch (Exception ex) {
+					continue;
+				}
+				result.add(gc);
+			}
+		}
```

Comment:
These could be reduced like so, eliminating the need for the temp var `gc`:
```diff
 		for(Litter litter : getLittersBirthed()) {
 			for(String id : litter.getOffspring()) {
 				try {
-					gc = Main.game.getNPCById(id));
+					result.add(Main.game.getNPCById(id));
 				} catch (Exception ex) {
 					continue;
 				}
-				result.add(gc);
 			}
 		}
```

Reply:
My bad! I was prototyping some more aggressive changes (ones that would impact other parts of the engine with minimal improvement to save times) that would have required a temporary variable and forgot to change it back. Pushing an update now.